src/bm_sim/lpm_trie.cpp

```diff
@@ -0,0 +1,294 @@
+/* Copyright 2013-present Barefoot Networks, Inc.
+ * Copyright 2021 VMware, Inc.
```

Comment:
@jafingerhut do you know if it would be ok to just use `Copyright 2025 P4lang Authors` here?

Reply:
As at least a reasonable placeholder for what I think it might end up being, perhaps `Copyright 2025 Contributors to the P4 Project`.  If we end up with something different, I think it will get changed later.